app/src/org/commcare/dalvik/application/CommCareApplication.java

```diff
@@ -179,27 +192,44 @@ public void onCreate() {
             Logger.registerLogger(new AndroidLogger(this.getGlobalStorage(AndroidLogEntry.STORAGE_KEY, AndroidLogEntry.class)));
             pil.dumpToNewLogger();
         }
-        
+
 //        PreferenceChangeListener listener = new PreferenceChangeListener(this);
 //        PreferenceManager.getDefaultSharedPreferences(this).registerOnSharedPreferenceChangeListener(listener);
-        
+
         intializeDefaultLocalizerData();
-        
-        //The fallback in case the db isn't installed 
+
+        //The fallback in case the db isn't installed
         resourceState = STATE_UNINSTALLED;
-        
+
         //We likely want to do this for all of the storage, this is just a way to deal with fixtures
-        //temporarily. 
+        //temporarily.
         //StorageManager.registerStorage("fixture", this.getStorage("fixture", FormInstance.class));
-        
+
 //        Logger.registerLogger(new AndroidLogger(CommCareApplication._().getStorage(AndroidLogEntry.STORAGE_KEY, AndroidLogEntry.class)));
-//        
+//
 //        //Dump any logs we've been keeping track of in memory to storage
 //        pil.dumpToNewLogger();
-        
+

```

Comment:
tab changes? these are clean, right? IE: tabs -> spaces, not spces -> tabs, right?